src/dpq2/value.d

```diff
@@ -22,16 +26,48 @@ struct Value
     this(ubyte[] data, in OidType oidType, bool isNull, in ValueFormat format = ValueFormat.BINARY) pure
     {
         this._data = data;
-        this.format = format;
-        this.oidType = oidType;
-        this.isNull = isNull;
+        this._format = format;
+        this._oidType = oidType;
+        this._isNull = isNull;
     }
 
     /// Null Value constructor
     this(in ValueFormat format, in OidType oidType) pure
     {
-        this.format = format;
-        this.oidType = oidType;
+        this._format = format;
+        this._oidType = oidType;
+    }
+
+    @safe const pure nothrow @nogc
```

Comment:
You sure what this really works for {} blocks?

Reply:
Yep.

You can try for example with `@nogc` removed there and:

```D
unittest
{
    import dpq2.conv.from_d_types;
    auto v = toValue("test");

    () @nogc
    {
        assert(!v.isNull);
    }();
}
```